tests/hooks/useFieldArray.test.js

```diff
@@ -0,0 +1,429 @@
+import { createRef } from 'react';
+import { mount } from 'enzyme';
+import { act, render } from '@testing-library/react';
+import FieldArrayTestForm from '../testUtils/FieldArrayTestForm';
+import testHook from '../testUtils/hookTestUtils';
+import logger from '../../src/utils/logger';
+import { useForm, useFieldArray } from '../../src';
+import * as inputTypeUtils from '../../src/utils/inputTypeUtils';
+
+describe('useFieldArray hook', () => {
+	afterEach(() => {
+		jest.clearAllMocks();
```

Comment:
Should be `jest.restoreAllMocks` to ensure the actual `react` is being restored
Actually, the `react` module will still not be available for other tests inside this file because every call to `jest.mock` is being yeeted to the top of the file at runtime; but we should still restore it just in case